configuration/configuration_test.go

```diff
@@ -72,6 +72,39 @@ func TestParseArgsMessage(t *testing.T) {
 	test.Equals(t, "ci-skip", configuration.WipCommitMessage)
 }
 
+func TestParseArgsStartRoom(t *testing.T) {
+	configuration := GetDefaultConfiguration()
+	test.Equals(t, configuration.WipBranchQualifier, "")
+
+	command, parameters, configuration := ParseArgs([]string{"mob", "start", "--room", "testroom"}, configuration)
+
+	test.Equals(t, "start", command)
+	test.Equals(t, "", strings.Join(parameters, ""))
+	test.Equals(t, "testroom", configuration.TimerRoom)
+}
+
+func TestParseArgsTimerRoom(t *testing.T) {
+	configuration := GetDefaultConfiguration()
+	test.Equals(t, configuration.WipBranchQualifier, "")
+
+	command, parameters, configuration := ParseArgs([]string{"mob", "timer", "10", "--room", "testroom"}, configuration)
+
+	test.Equals(t, "timer", command)
+	test.Equals(t, "10", strings.Join(parameters, ""))
+	test.Equals(t, "testroom", configuration.TimerRoom)
+}
+
+func TestParseArgsTimerOpenRoom(t *testing.T) {
+	configuration := GetDefaultConfiguration()
+	test.Equals(t, configuration.WipBranchQualifier, "")
+
+	command, parameters, configuration := ParseArgs([]string{"mob", "timer", "open", "--room", "testroom"}, configuration)
+
+	test.Equals(t, "timer", command)
+	test.Equals(t, "open", strings.Join(parameters, ""))
+	test.Equals(t, "testroom", configuration.TimerRoom)
+}
+
```

Comment:
nice, lets also test that the arg is doing what its supposed to be doing

Reply:
I would not extend the test here to check if the timer works. As this is just a test which should check if the configuration works. We have a test in `mob_test.go` which checks it the browser would open up, see `mob_test.go:1726`:
```go
func TestOpenTimerInBrowserWithTimerRoom(t *testing.T) {
	mockOpenInBrowser()
	output, configuration := setup(t)
	configuration.TimerRoom = "testroom"

	err := openTimerInBrowser(configuration)

	assertOutputNotContains(t, output, "Timer Room is not configured.")
	assertNoError(t, err)
}
```